variants/rak4631/variant.h

```diff
@@ -133,6 +133,24 @@ static const uint8_t SCK = PIN_SPI_SCK;
 #define PIN_EINK_SCLK (0 + 3)
 #define PIN_EINK_MOSI (0 + 30) // also called SDI
 
+// E-Ink model selection
+
+#define EINK_DISPLAY_MODEL GxEPD2_213_BN // 2.13 inch b/w 250x122
+#define EINK_WIDTH 250
+#define EINK_HEIGHT 122
+
+// #define EINK_DISPLAY_MODEL GxEPD2_420_M01 // 4.2 inch 300x400
```

Comment:
In cases like this, would you like me to bring the commented-out boards into the platformio.ini file also, as grouped sets of build flags?

```ini
build_flags = 
  ; 2.13 inch b/w 250x122 
  -DEINK_DISPLAY_MODEL=GxEPD2_213_BN
  -DEPD_WIDTH=250
  -DEPD_HEIGHT=122

  ; 4.2 inch 300x400
  ;-DEINK_DISPLAY_MODEL=GxEPD2_420_M01
  ;-DEPD_WIDTH=300
  ;-DEPD_HEIGHT=400
```
(etc)

Reply:
No we can consolidate those just into a single list to keep as known tested or used displays variants